src/main/java/io/confluent/connect/elasticsearch/Mapping.java

```diff
@@ -47,14 +47,6 @@
 
   private static final Logger log = LoggerFactory.getLogger(Mapping.class);
 
-  /**
-   * Create an explicit mapping.
-   * @param client The client to connect to Elasticsearch.
-   * @param index The index to write to Elasticsearch.
-   * @param type The type to create mapping for.
-   * @param schema The schema used to infer mapping.
-   * @throws IOException
-   */
```

Comment:
I removed the Javadoc because they did not add any information beyond what the method and parameter names already conveyed. Since this is implementation rather than library code, I think it's reasonable to omit docs in such cases.